build/windows/sail.wxs

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<?ifndef ProductVersion?>
+<?error ProductVersion property not defined?>
+<?endif?>
+
+<!-- Define a unique UpgradeCode per platform -->
+<?if $(var.Platform) = "x64"?>
+<?define InstallerVersion = "200"?>
+<?define UpgradeCode = "8CFB9531-B959-4E1B-AA2E-4AF0FFCC4AF4"?>
+<?define ProgramFilesFolder = "ProgramFiles64Folder"?>
+<?elseif $(var.Platform) = "x86"?>
+<?define InstallerVersion = "200"?>
+<?define UpgradeCode = "767EC5D2-C8F0-4912-9901-45E21F59A284"?>
+<?define ProgramFilesFolder = "ProgramFilesFolder"?>
+<?elseif $(var.Platform) = "arm64"?>
+<?define InstallerVersion = "500"?>
+<?define UpgradeCode = "5D15E95C-F979-41B0-826C-C33C8CB5A7EB"?>
+<?define ProgramFilesFolder = "ProgramFiles64Folder"?>
+<?elseif $(var.Platform) = "arm"?>
+<?define InstallerVersion = "500"?>
+<?define UpgradeCode = "DDDE52AA-42DA-404B-9238-77DC86117CFF"?>
+<?define ProgramFilesFolder = "ProgramFilesFolder"?>
+<?endif?>
+
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+    <Product Id="*" Name="SailPoint CLI" Version="$(var.ProductVersion)" Language="1033"
+        Manufacturer="SailPoint Technologies." UpgradeCode="$(var.UpgradeCode)">
+        <Package Compressed="yes" InstallerVersion="$(var.InstallerVersion)"
+            InstallScope="perMachine" />
+        <MediaTemplate EmbedCab="yes" />
+
+        <!-- Remove older product(s) early but within the transaction -->
+        <MajorUpgrade Schedule="afterInstallInitialize"
+            DowngradeErrorMessage="A newer version of !(bind.property.ProductName) is already installed." />
+
+        <!-- Upgrade older x86 products -->
+        <Upgrade Id="7C0A5736-5B8E-4176-B350-613FA2D8A1B3">
```

Comment:
Is this ID somehow specific to our installation? Maybe this isn't even needed, not sure

Reply:
Oh Jeez, I really have no idea, Maybe thats something we need to generate?

Reply:
https://stackoverflow.com/questions/23529029/wix-generating-new-upgrade-code

Any chance you could help me make sense of all this?

Reply:
Ok so these values are GUIDs for different versions of your app on different platforms like arm, amd64 stuff like that, Do you know how we can generate those GUIDs?